compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/SwingComposeSceneLayer.desktop.kt

```diff
@@ -71,18 +71,14 @@ internal class SwingComposeSceneLayer(
         it.background = Color.Transparent.toAwtColor()
         it.size = Dimension(windowContainer.width, windowContainer.height)
         it.addMouseListener(backgroundMouseListener)
-
-        // TODO: Currently it works only with offscreen rendering
-        // TODO: Do not clip this from main scene if layersContainer == main container
-        windowContainer.add(it, JLayeredPane.POPUP_LAYER, 0)
     }
 
     private var containerSize = IntSize.Zero
         set(value) {
             if (field.width != value.width || field.height != value.height) {
                 field = value
                 container.setBounds(0, 0, value.width, value.height)
-                if (_boundsInWindow == null) {
+                if (drawBounds.isEmpty) {
```

Comment:
Does this intend to check whether the drawBounds have not been set yet, or whether nothing was drawn?

Reply:
not initialized. Removed - not required anymore